src/vstart.sh

```diff
@@ -732,6 +734,12 @@ prepare_conf() {
         msgr_conf="ms bind msgr2 = false
                    ms bind msgr1 = true"
     fi
+    if [ $force_addr -eq 1 ]; then
+        msgr_conf+="
+                    public bind addr = $IP
+                    public addr = $IP
+                    cluster addr = $IP"
+    fi
```

Comment:
Try:

```
diff --git a/src/vstart.sh b/src/vstart.sh
index d91c2d70331..1d54f28185e 100755
--- a/src/vstart.sh
+++ b/src/vstart.sh
@@ -742,9 +742,11 @@ prepare_conf() {
     if [ $force_addr -eq 1 ]; then
         msgr_conf+="
                     public bind addr = $IP
-                    public addr = $IP
-                    cluster addr = $IP"
+                    public addr = $IP"
     fi
+    msgr_conf+="
+                cluster addr = 127.0.0.1
+                cluster network = 127.0.0.1/8"
 
     wconf <<EOF
 ; generated by vstart.sh on `date`
```

I think we can hard-code `cluster addr` and `cluster network` to always use loopback.

Reply:
done

Reply:
The API test failure may be related to this, please check.

Reply:
yeah, that's likely. I'll revert this for now.